app/src/main/java/vn/ztech/software/movie_streaming/data/model/Episode.kt

```diff
@@ -0,0 +1,9 @@
+package vn.ztech.software.movie_streaming.data.model
+
+data class Episode(
+    val id: String,
+    val number: Int,
+    val season: Int,
+    val title: String,
+    val url: String
```

Comment:
```suggestion
data class Episode(
    val id: String?,
    val number: Int?,
    val season: Int?,
    val title: String?,
    val url: String?
```
anh check những file tương tự luôn nhé, cái này để tránh data bị null